tools/export/qtcreator/creator.tmpl

```diff
@@ -0,0 +1 @@
+[General]
```

Comment:
Hahaha I told you that I was a bit ashamed to inconvenience Jinja just for this kind of output. :smile: